multi/stm32l4-multi/stm32l4-multi.c

```diff
@@ -244,6 +245,7 @@ int main(void)
 	portCreate(&common.port);
 #endif
 
+	fs_init();
```

Comment:
Move this after line 262 - in case `CONSOLE_IS_TTY` you would create fs port before console port